setup.ts

```diff
@@ -45,113 +44,6 @@ async function accessAndRefreshTokens(
   }
 }
 
-// Check connection to Redis with the specified URL.
```

Comment:
1. There is no need to have this file included in this PR
1. You have also removed key functionality in it.
1. Exclude this file from the PR
1. Merge with the latest upstream also